starter/pom.xml

```diff
@@ -156,6 +156,20 @@
                     </to>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <argLine>--add-opens java.base/java.util=ALL-UNNAMED --add-opens java.base/java.lang=ALL-UNNAMED</argLine>
```

Comment:
Is this solution backward-compatible, i.e. does this cause any issues when used with older JREs?
Also this seems to be overriding the command-line parameters passed via `-Dxxx=yyy` (used here https://github.com/FraunhoferIOSB/FAAAST-Service/blob/c982cc7ef14fa30f5f7a0e1d649fab111668958b/.github/workflows/maven-build-pr.yml#L51) which breaks the CI pipeline. 
Is there any other way to do this?